### ISSUE TITLE
Mesh won't connect after upgrade - Auth error

### ISSUE THREAD

**Issue by JoeyGnarf** [fsaris/home-assistant-awox#65]
After upgrading HA and this AwoX intergration I can't seem to connect to the mesh anymore. In the logs it shows me
`2022-09-22 09:02:56.869 INFO (SyncWorker_1) [custom_components.awox.awoxmeshlight] Auth error : check name and password.`

To try and resolve this I've:

1. deleted the integration and deleted the files from the custom_components folder, just to be sure
2. in the Smart Control app I've deleted all lights, changed my account password (just to be sure there was no conflict with the newer AwoX app)
3. rebooted Home Assitant
4. added all the lights in the Smart Control app
5. reinstalled the integration and logged in with the new credentials
6. rebooted yet another time

But all to no avail...

I've had this same issue (Auth error) with a previous HA upgrade (> 2021.12) as well, but simply reverted to a backup since I hadn't the time to look into it.

Can you advice?

**Comment by fsaris** [comment]
@JoeyGnarf that message implies that the used info fetched from Awox when you setup the integration is outdated. Did you use the same credentials in HA when you re-installed the integration as you used to login to the Awox app?

_note: Will improve the logged message_

**Comment by JoeyGnarf** [comment]
Yes. Even removed all lights and remotes from the app, manually reset all of them, power cycled the entire house, created a new account with different credentials, added them to the new account, clean installed the integration and I still get the same thing :-(

PS. Thanks for your help and for your work on updating the integration.

**Comment by fsaris** [comment]
If you change the name of one of the lights in the Awox app and then reinstall the integration in HA is this new name visible in HA?

Could you enable debug logging before you install the integration and share that log with me?

**Comment by JoeyGnarf** [comment]
I'm afraid I can only let you know in a few months. I was so fed up I've replaced all AwoX BT controlled lights and stored those to try again later if and when we can use them again in a new karate studio we're hoping to open in January. 

Again, many thanks for your efforts and support while I had them in use!
